Gemfile

```diff
@@ -18,6 +18,7 @@ end
 group :test do
   gem 'spy'
   gem 'mocha'
+  gem 'timecop'
```

Comment:
Instead of adding timecop it would be preferable to use Rails time helpers: https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to

Reply:
nice. I wasn't aware that this existed :)